core/health/impl/src/main/java/org/opennms/core/health/impl/DefaultHealthCheckService.java

```diff
@@ -41,12 +41,7 @@
 import java.util.function.Consumer;
 import java.util.stream.Collectors;
 
-import org.opennms.core.health.api.Context;
-import org.opennms.core.health.api.Health;
-import org.opennms.core.health.api.HealthCheck;
-import org.opennms.core.health.api.HealthCheckService;
-import org.opennms.core.health.api.Response;
-import org.opennms.core.health.api.Status;
+import org.opennms.core.health.api.*;
```

Comment:
In general, use specific imports instead of wildcard.

Reply:
updated